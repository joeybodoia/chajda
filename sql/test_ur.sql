CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"');
