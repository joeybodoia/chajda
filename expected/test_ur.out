CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                     spacy_lemmatize                                                     
-------------------------------------------------------------------------------------------------------------------------
   میرا:3 مثال:4 آزمائشی:5 جملہ:6 جو:8 میں:9 گوگل:10 ترجمہ:12 میں:13 ٹیسٹ:14 معاملات:16 پیدا:17 کرنا:18 ڈالنا:21 رہنا:22
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                 spacy_lemmatize                                  
----------------------------------------------------------------------------------
   میرا مثال آزمائشی جملہ جو میں گوگل ترجمہ میں ٹیسٹ معاملات پیدا کرنا ڈالنا رہنا
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                            spacy_lemmatize                                                                            
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
   میں:2 میرا:3 مثال:4 آزمائشی:5 جملہ:6 ہونا:7 جو:8 میں:9 گوگل:10 کم:11 ترجمہ:12 میں:13 ٹیسٹ:14 کم:15 معاملات:16 پیدا:17 کرنا:18 کم:19 لئے:20 ڈالنا:21 رہنا:22 ہونا:23
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                               spacy_lemmatize                                               
-------------------------------------------------------------------------------------------------------------
   میں میرا مثال آزمائشی جملہ ہونا جو میں گوگل کم ترجمہ میں ٹیسٹ کم معاملات پیدا کرنا کم لئے ڈالنا رہنا ہونا
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                            spacy_lemmatize                                                             
----------------------------------------------------------------------------------------------------------------------------------------
   ":2 میرا:4 مثال:5 آزمائشی:6 جملہ:7 جو:9 میں:10 گوگل:11 ترجمہ:13 میں:14 ٹیسٹ:15 معاملات:17 پیدا:18 کرنا:19 ڈالنا:22 رہنا:23 ۔:25 ":26
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                    spacy_lemmatize                                     
----------------------------------------------------------------------------------------
   " میرا مثال آزمائشی جملہ جو میں گوگل ترجمہ میں ٹیسٹ معاملات پیدا کرنا ڈالنا رہنا ۔ "
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                                   spacy_lemmatize                                                                                    
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 میں:3 میرا:4 مثال:5 آزمائشی:6 جملہ:7 ہونا:8 جو:9 میں:10 گوگل:11 کم:12 ترجمہ:13 میں:14 ٹیسٹ:15 کم:16 معاملات:17 پیدا:18 کرنا:19 کم:20 لئے:21 ڈالنا:22 رہنا:23 ہونا:24 ۔:25 ":26
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   " میں میرا مثال آزمائشی جملہ ہونا جو میں گوگل کم ترجمہ میں ٹیسٹ کم معاملات پیدا کرنا کم لئے ڈالنا رہنا ہونا ۔ "
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                                     spacy_lemmatize                                                     
-------------------------------------------------------------------------------------------------------------------------
   میرا:3 مثال:4 آزمائشی:5 جملہ:6 جو:8 میں:9 گوگل:10 ترجمہ:12 میں:13 ٹیسٹ:14 معاملات:16 پیدا:17 کرنا:18 ڈالنا:21 رہنا:22
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                                 spacy_lemmatize                                  
----------------------------------------------------------------------------------
   میرا مثال آزمائشی جملہ جو میں گوگل ترجمہ میں ٹیسٹ معاملات پیدا کرنا ڈالنا رہنا
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                                            spacy_lemmatize                                                                            
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------
   میں:2 میرا:3 مثال:4 آزمائشی:5 جملہ:6 ہونا:7 جو:8 میں:9 گوگل:10 کم:11 ترجمہ:12 میں:13 ٹیسٹ:14 کم:15 معاملات:16 پیدا:17 کرنا:18 کم:19 لئے:20 ڈالنا:21 رہنا:22 ہونا:23
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                               spacy_lemmatize                                               
-------------------------------------------------------------------------------------------------------------
   میں میرا مثال آزمائشی جملہ ہونا جو میں گوگل کم ترجمہ میں ٹیسٹ کم معاملات پیدا کرنا کم لئے ڈالنا رہنا ہونا
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                                            spacy_lemmatize                                                             
----------------------------------------------------------------------------------------------------------------------------------------
   ":2 میرا:4 مثال:5 آزمائشی:6 جملہ:7 جو:9 میں:10 گوگل:11 ترجمہ:13 میں:14 ٹیسٹ:15 معاملات:17 پیدا:18 کرنا:19 ڈالنا:22 رہنا:23 ۔:25 ":26
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                                    spacy_lemmatize                                     
----------------------------------------------------------------------------------------
   " میرا مثال آزمائشی جملہ جو میں گوگل ترجمہ میں ٹیسٹ معاملات پیدا کرنا ڈالنا رہنا ۔ "
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                                                   spacy_lemmatize                                                                                    
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
   ":2 میں:3 میرا:4 مثال:5 آزمائشی:6 جملہ:7 ہونا:8 جو:9 میں:10 گوگل:11 کم:12 ترجمہ:13 میں:14 ٹیسٹ:15 کم:16 معاملات:17 پیدا:18 کرنا:19 کم:20 لئے:21 ڈالنا:22 رہنا:23 ہونا:24 ۔:25 ":26
(1 row)

SELECT spacy_lemmatize('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                                  spacy_lemmatize                                                  
-------------------------------------------------------------------------------------------------------------------
   " میں میرا مثال آزمائشی جملہ ہونا جو میں گوگل کم ترجمہ میں ٹیسٹ کم معاملات پیدا کرنا کم لئے ڈالنا رہنا ہونا ۔ "
(1 row)

SELECT spacy_tsvector('ur',' "یہ میرا مثال آزمائشی جملہ ہے جو میں گوگل کے ترجمے میں ٹیسٹ کے معاملات پیدا کرنے کے لئے ڈال رہا ہوں۔"');
                                                                spacy_tsvector                                                                 
-----------------------------------------------------------------------------------------------------------------------------------------------
 'آزمائشی':5 'ترجمہ':12 'جملہ':6 'جو':8 'رہنا':22 'مثال':4 'معاملات':16 'میرا':3 'میں':9,13 'ٹیسٹ':14 'پیدا':17 'ڈالنا':21 'کرنا':18 'گوگل':10
(1 row)

